src/components/NewToDo/NewToDo.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './NewToDo.css';
+
+class NewToDo extends Component {
+  state = {
+    newTask: '',
```

Comment:
```suggestion
    title: '',
```
`newTask` подразумевает готовый объект

Reply:
> `newTask` подразумевает готовый объект

refactored